scrapy_zyte_smartproxy/middleware.py

```diff
@@ -13,6 +13,9 @@
 from scrapy_zyte_smartproxy.utils import exp_backoff
 
 
+logger = logging.getLogger("zyte-smart-proxy-manager")
```

Comment:
oh sorry I just saw this again, and any special reason for this name? I think it would be easier to remember the package name `scrapy-zyte-smartproxy`

Reply:
I have no strong preference about the name, as long as it's not the root logger 😄 
I'll make the change.

Reply:
Updated

Reply:
great ty 👍